modules/ROOT/examples/live-demos/typography/index.js

```diff
@@ -0,0 +1,81 @@
+tinymce.init({
+  selector: "textarea#advanced-typography",
+  height: 300,
+  menu: { tools: { title: 'Tools', items: 'code wordcount typography' }},
+  plugins: 'code visualchars nonbreaking typography wordcount',
+  toolbar: 'bold italic | code visualchars nonbreaking | typography language',
+  content_langs: [
+    { title: 'English', code: 'en' },
+    { title: 'Spanish', code: 'es' },
+    { title: 'German', code: 'de' }
+  ],
+  content_style: 'body { font-family:Helvetica,Arial,sans-serif; font-size:16px }',
+  typography_default_lang: 'en-US',
+  typography_langs: [
+    'en-US',
+    'es',
+    'de'
+  ],
+  typography_rules: [
+    'common/punctuation/quote',
+    'en-US/dash/main',
+    'common/nbsp/afterParagraphMark',
+    'common/nbsp/afterSectionMark',
+    'common/nbsp/afterShortWord',
+    'common/nbsp/beforeShortLastNumber',
+    'common/nbsp/beforeShortLastWord',
+    'common/nbsp/dpi',
+    'common/punctuation/apostrophe',
+    'common/space/delBeforePunctuation',
+    'common/space/afterComma',
+    'common/space/afterColon',
+    'common/space/afterExclamationMark',
+    'common/space/afterQuestionMark',
+    'common/space/afterSemicolon',
+    'common/space/beforeBracket',
+    'common/space/bracket',
+    'common/space/delBeforeDot',
+    'common/space/squareBracket',
+    'common/number/mathSigns',
+    'common/number/times',
+    'common/number/fraction',
+    'common/symbols/arrow',
+    'common/symbols/cf',
+    'common/symbols/copy',
+    'common/punctuation/delDoublePunctuation',
+    'common/punctuation/hellip'
+  ],
+  typography_ignore: [ 'code' ]
+});
+
+var languages = {
+  'et': 'Estonian',
+  'eo': 'Esperanto',
+  'en-US': 'English (US)',
+  'en-GB': 'English (UK)',
+  'bg': 'Bulgarian',
+  'be': 'Belarusian',
+  'ca': 'Catalan',
+  'cs': 'Czech',
+  'ga': 'Irish',
+  'da': 'Danish',
+  'nl': 'Dutch',
+  'fi': 'Finnish',
+  'fr': 'French',
```

Comment:
Should we also temporarily remove French from here?

Reply:
This PR was closed in favour of https://github.com/tinymce/tinymce-docs/pull/2578, so you'll need to review that one instead.